syft_tensorflow/hook/hook.py

```diff
@@ -284,7 +284,7 @@ def _add_methods_from_native_tensor(tensor_type: type, syft_type: type):
 
     @classmethod
     def create_wrapper(cls, child_to_wrap, *args, **kwargs):
-        if isinstance(child_to_wrap.object_type, tf.Variable):
+        if type(child_to_wrap.object_type)==type(tf.Variable):
```

Comment:
Why this change?

Reply:
it's because `child_to_wrap.object_type` is not instantiated so it's like doing `isinstance(tf.Variable, tf.Variable)` and it will return False. 

child_to_wrap.object_type is defined [here](https://github.com/OpenMined/PySyft-TensorFlow/blob/c8cee90667eb8ea41efda46faf25307411f22537/syft_tensorflow/tensor/variable.py#L236)

do you have another solution?

Reply:
In that case the type wrapper is unnecessary, you can just do `if child_to_wrap.object_type == tf.Variable`

Reply:
fixed, let's forget about this question haha. thanks